meta/main.yml

```diff
@@ -17,13 +17,13 @@ galaxy_info:
     - name: Debian
       versions: [bullseye]
     - name: EL
-      versions: ['7', '8', '9']
+      versions: ["7", "8", "9"]
```

Comment:
Can you change double quotes here back to single quotes (I use single quotes for array values)? (Same for the other array/list values in the metadata.)